app/src/main/java/com/android/shelfLife/model/user/UserRepository.kt

```diff
@@ -33,12 +30,6 @@ interface UserRepository {
    */
   fun stopListeningForInvitations()
 
-  /**
-   * Sets the user's logged-in status. This is used to determine whether to show the sign-in screen
-   *
-   * @param isLoggedIn - The new logged-in status.
-   */
-  fun setUserLoggedInStatus(isLoggedIn: Boolean)
```

Comment:
as the "userLoggedInStatus" is no longer part of the repository, we do not need this function